Makefile

```diff
@@ -333,18 +333,19 @@ GOPATH=$(shell go env GOPATH)
 	cd "${GOPATH}/src/github.com/golang/mock/mockgen" && git checkout 1.3.1 && go get ./... && go install ./... && cd -
 	go get golang.org/x/tools/cmd/goimports
 	GO111MODULE=on go get github.com/fzipp/gocyclo/cmd/gocyclo@v0.3.1
-	go get honnef.co/go/tools/cmd/staticcheck
+	GO111MODULE=on go get honnef.co/go/tools/cmd/staticcheck@v0.3.1
 	touch .get-deps-stamp
 
 get-deps: .get-deps-stamp
 
 get-deps-init:
 	go get golang.org/x/tools/cover
 	go get golang.org/x/tools/cmd/cover
+	go get github.com/golang/mock/mockgen
+	cd "${GOPATH}/src/github.com/golang/mock/mockgen" && git checkout 1.3.1 && go get ./... && go install ./... && cd -
 	GO111MODULE=on go get github.com/fzipp/gocyclo/cmd/gocyclo@v0.3.1
 	go get golang.org/x/tools/cmd/goimports
-	go get github.com/golang/mock/mockgen
-	go get honnef.co/go/tools/cmd/staticcheck
+	GO111MODULE=on go get honnef.co/go/tools/cmd/staticcheck@v0.3.1
```

Comment:
nit: We have v0.3.2 in master, is it okay to have that here too?

Reply:
Nah, it breaks the static checks if I try with 0.3.2.